homeassistant/components/tahoma_api.py

```diff
@@ -0,0 +1,1155 @@
+"""
```

Comment:
This file needs to become a module which is published on PyPI and then added as [requirement](https://home-assistant.io/developers/code_review_platform/#1-requirements). See https://jeffknupp.com/blog/2013/08/16/open-sourcing-a-python-project-the-right-way/